marketplace/deployer_helm_tiller_base/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM launcher.gcr.io/google/debian9
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    bash \
+    ca-certificates \
+    curl \
+    gettext \
+    git \
+    jq \
+    procps \
+    python \
+    python-pip \
+    python-setuptools \
+    python-yaml \
+    wget \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN pip install wheel \
+    && pip install google-cloud-storage
+
+RUN wget -q -O /bin/kubectl \
+        https://storage.googleapis.com/kubernetes-release/release/v1.9.0/bin/linux/amd64/kubectl \
+    && chmod 755 /bin/kubectl
+
+RUN mkdir -p /bin/helm-downloaded \
+    && wget -q -O /bin/helm-downloaded/helm.tar.gz \
+        https://storage.googleapis.com/kubernetes-helm/helm-v2.10.0-linux-amd64.tar.gz \
+    && tar -zxvf /bin/helm-downloaded/helm.tar.gz -C /bin/helm-downloaded \
+    && mv /bin/helm-downloaded/linux-amd64/helm /bin/ \
+    && rm -rf /bin/helm-downloaded \
+    && helm init --client-only \
+    && helm plugin install https://github.com/rimusz/helm-tiller \
+    && helm plugin install https://github.com/nouney/helm-gcs \
+    && helm repo add cloud-marketplace-tools gs://cloud-marketplace-tools-charts/
+
+COPY marketplace/deployer_util/* /bin/
+COPY marketplace/deployer_helm_tiller_base/bin/* /bin/
+
+RUN mkdir -p /data/chart
+
+ENTRYPOINT ["/bin/bash", "/bin/deploy.sh"]
```

Comment:
I'd like to introduce an `entrypoint.sh` similar to the `dev` image that will do `exec "$@"` at the end. It's a nice paradigm to follow, to avoid having to change entrypoint of the docker container--only args need specifying. 

As long as that `entrypoint.sh` does do any required setup, we can add it now. It's backward compatible with the way our test run modifies the entrypoint.

Would suggest at least dropping `/bin/bash`, having this only as `ENTRYPOINT ["/bin/deploy.sh"]`. `/bin/bash` doesn't add much value; I find it more confusing to use, like one would wonder why specifying `echo "Hello world"` to the docker run command doesn't work.

Reply:
I'd prefer to do this as a follow up - it's not partner facing.